### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by korhan-square**

### CODE REVIEW

**Comment by korhan-square** [comment]
test1

**Comment by petrucci34** [comment]
test

**Comment by petrucci34** [comment]
test1

**Comment by korhan-square** [comment]
test2

**Comment by petrucci34** [comment]
test3

**Comment by korhan-square** [comment]
- [x] hello 2

**Comment by korhan-square** [comment]
test3

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # iOS-test repo readme
+test5
```
Comment (korhan-square) [inline README.md:2]: test

**Review by korhan-square — COMMENTED** [review]
comment

File: README.md (hunk 1 above)
Comment (korhan-square) [reply README.md:2]: - [x] first
- [x] second

Comment (korhan-square) [reply README.md:2]: - [x] third

Comment (korhan-square) [reply README.md:2]: - [x] four

Comment (korhan-square) [reply README.md:2]: - [x] five

Comment (korhan-square) [reply README.md:2]: - [x] six

**Review by korhan-square — COMMENTED** [review]
comment

File: README.md (hunk 1 above)
Comment (korhan-square) [reply README.md:2]: - [x] seven

**Pull request closed without merging**